tests/installation/resolve_dependency_issues.pm

```diff
@@ -33,8 +33,12 @@ sub run {
 
 sub post_fail_hook {
     my $self = shift;
-    select_console 'root-console';
-    $self->upload_solvertestcase_logs();
+    select_console 'log-console';
+    if (check_var('UPGRADE', '1')) {
```

Comment:
Hajian, 
If my understanding is correct, we may not be able to know when the issue happens, so why not uploading this file by default?

Reply:
zypper is not available when we exit from iso installation/update